config/locales/views/schools/schools.yml

```diff
@@ -326,6 +326,17 @@ en:
       audit:
         message_html: You have completed <strong>%{completed_activities_count}/%{total_activities_count}</strong> of the activities and <strong>%{completed_actions_count}/%{total_actions_count}</strong> of the actions from your recent energy audit
         summary_html: Complete the others to score <strong>%{remaining_points}</strong> points and <strong>%{bonus_points}</strong> bonus points for completing all audit tasks
+      completed_programme:
+        button: View programme
+        message_html: Well done, you've just completed the <strong>%{title}</strong> programme and have earned <strong>%{bonus_points}</strong> bonus points!
```

Comment:
One more comment: what if they've completed the programme but are not eligible for the bonus as it wasn't done in same academic year. Should we have different text to avoid saying "earned 0 bonus points"